src/test/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaDataTest.java

```diff
@@ -591,4 +595,49 @@ void testGetImportedKeysArrayWithWhiteSpaces() throws SQLException {
         final ResultSet arrayImportedKeys = metadata.getImportedKeys("", "", COLLECTION_ARRAY + "_array");
         Assertions.assertFalse(arrayImportedKeys.next());
     }
+
+    @Test
+    @DisplayName("Tests getting type information.")
+    void testGetTypeInfo() throws SQLException {
+        final List<Integer> expectedDataTypes = Arrays.asList(
+                Types.BINARY,
+                Types.BIGINT,
+                Types.BOOLEAN,
+                Types.CHAR,
+                Types.DATE,
+                Types.DECIMAL,
+                Types.DOUBLE,
+                Types.FLOAT,
+                Types.INTEGER,
+                Types.NCHAR,
+                Types.NULL,
+                Types.NVARCHAR,
+                Types.REAL,
+                Types.SMALLINT,
+                Types.TIME,
+                Types.TIMESTAMP,
+                Types.TINYINT,
+                Types.VARBINARY,
+                Types.VARCHAR
+        ).stream().sorted().collect(Collectors.toList());
+        final List<Integer> observedDataTypes = new ArrayList<>();
+
+        final ResultSet resultSet = metadata.getTypeInfo();
+        while (resultSet.next()) {
+            final int jdbcDataType = resultSet.getInt(2);
+            observedDataTypes.add(jdbcDataType);
+            if (expectedDataTypes.contains(jdbcDataType)) {
+                Assertions.assertEquals(JdbcType.fromType(jdbcDataType).name(), resultSet.getString(1));
+            } else {
+                Assertions.fail(String.format(
+                        "Unexpected type returned %d-'%s'",
+                        jdbcDataType,
+                        JdbcType.fromType(jdbcDataType).name()));
+            }
+        }
+        final List<Integer> sortedDataTypes = observedDataTypes.stream()
+                .sorted()
+                .collect(Collectors.toList());
```

Comment:
just wondering, does the format need to be fixed here (pad more indentation) or is it normal?